### PR TITLE
Feature/use 15 min step

### PR DESCRIPTION
**Pull request by flibbertigibbet**
- Add time surface tile layer. Takes awhile for cache to warm, but it's snappy after that.
- Fetch isochrones individually, as geometries break when requesting multiple (see [this issue](opentripplanner/OpenTripPlanner#1436))
- Change slider step to 15 minute increments
- Fix bar graph display, using new results from histogram
- Disable point-in-polygon check temporarily, for responsiveness. Will add travel times to each point in server response to use instead for styling.

### CODE REVIEW

**Comment by flibbertigibbet** [comment]
isochrones-overlays-15-min

File: js/OTPALayer.js [hunk 1]
```diff
@@ -91,16 +91,29 @@ L.OTPALayer = L.FeatureGroup.extend({
...
           fillColor: '#333',
           lineCap: 'round',
           lineJoin: 'round',
           weight: 2,
-          dashArray: '5, 4',
-          fillOpacity: '0.08'
+          //dashArray: '5, 4',
+          fillOpacity: '0.1',
+          smoothFactor: 0.25,
+          noClip: true,
```
Comment (maurizi) [inline js/OTPALayer.js]: Looks like you've specified `smoothFactor` and `noClip` twice here.

**Comment by maurizi** [comment]
Nice!
Not sure its a huge problem, but the polygonal layer controlled by the slider doesn't match up with the underlying tile layer.  That plus the subtle styling on the polygonal layer made me confused, originally I thought the slider wasn't working.  Maybe bringing back the dashed lines would help with the latter problem, but I don't know what to do about it not matching the til layer

**Comment by flibbertigibbet** [comment]
Right, the weirdnesses in the isochrone shapes fetched as geojson were what led me to put in the surface tile layer, which is accurate; also the travel times returned by the analyst processing seem to be pretty accurate and line up well with the surface tiles. It was nice to have them presented together to sort that out, but now I think maybe we should just ditch the geojson isochrones altogether.

I've got the travel times for each point coming back with the indicators now, so we can maybe do some fancy point styling based on that for a visual cue to go with the slider.

File: js/OTPALayer.js (hunk 1 above)
Comment (flibbertigibbet) [inline js/OTPALayer.js]: Yup, I sure did. Neither is really helpful; I wanted to make sure that what I was seeing wasn't Leaflet messing with things by simplifying geometries (that wasn't it). I'll take them both out.

**Comment by maurizi** [comment]
Looks good!  I'll branch off of this and resurrect the different styling of the points within the travel shed based off of the server-side computed travel times (and probably kill the polygon layer completely).

**Comment by flibbertigibbet** [comment]
Okay, great! I'll go ahead and merge this in then.

**Pull request merged**